sourcetool/pkg/policy/policy.go

```diff
@@ -225,7 +225,7 @@ func CreateLocalPolicy(ctx context.Context, ghconnection *ghcontrol.GitHubConnec
 	}
 
 	p := RepoPolicy{
-		CanonicalRepo: "TODO fill this in",
+		CanonicalRepo: fmt.Sprintf("https://github.com/%s/%s", ghconnection.Owner(), ghconnection.Repo()),
```

Comment:
FWIW I've thought about removing this field.  I'm not sure it provides any value (and it's not required anymore).

No action needed (but let me know if you like it).

Reply:
I like having the repo name there to ensure the document can stand alone, if we don't have it then the only way to link it back to the repo is the path where it's stored. I'll leave it for now and let's discuss if we drop the field.